### PR TITLE
Feature/em 15 integrate code climate

### PR DESCRIPTION
**Pull request by mrubi-rootstrap**
## What does this PR do?

It integrates [CodeClimate into the project view](https://rootstrap.atlassian.net/jira/software/projects/EM/boards/42?selectedIssue=EM-15) 

It does not implement the population of the CodeClimate data which is to be implemented in a different task

### CODE REVIEW

File: db/structure.sql [hunk 1]
```diff
@@ -106,7 +106,7 @@ CREATE TYPE public.review_state AS ENUM (
 
 SET default_tablespace = '';
 
-SET default_table_access_method = heap;
+SET default_with_oids = false;
```
Comment (hdamico) [inline db/structure.sql:109]: What does this do?

File: app/services/metrics/code_climate_summary_retriever.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+module Metrics
+  class CodeClimateSummaryRetriever < BaseService
+    attr_reader :project_id
+
+    def initialize(project_id)
+      @project_id = project_id
+    end
+
+    def call
+      CodeClimateProjectMetric.joins(:project).where(projects: { id: project_id }).first
```
Comment (hdamico) [inline app/services/metrics/code_climate_summary_retriever.rb:10]: Could we use `find_by` right?
`CodeClimateProjectMetric.joins(:project).find_by(projects: { id: project_id })`

File: db/structure.sql (hunk 1 above)
Comment (mrubi-rootstrap) [reply db/structure.sql:109]: I couldn't tell. This file is generated by Rails every time I recreate the migrations.

Comment (hdamico) [reply db/structure.sql:109]: I am on your branch and I ran the migrations on my local and I don't have that 🤔 
image

File: app/services/metrics/code_climate_summary_retriever.rb (hunk 2 above)
Comment (mrubi-rootstrap) [reply app/services/metrics/code_climate_summary_retriever.rb]: Indeed.
Fixed it. Thanks!

File: db/structure.sql (hunk 1 above)
Comment (horacio) [reply db/structure.sql:109]: This is most likely a DB version thing.

@mrubi-rootstrap which version of Postgres do you have?

Comment (mrubi-rootstrap) [reply db/structure.sql:109]: It seems like a Postgres setting, yes.
I am using postgres:11 in a docker container, will look for setting that value to match the previous value

Comment (mrubi-rootstrap) [reply db/structure.sql]: In the mean time I reverted it to its previous value

**Pull request merged**